build/Dockerfile

```diff
@@ -1,14 +1,12 @@
-FROM fedora:30
+FROM fedora:31
 
-RUN sudo dnf install -y nmstate iproute iputils && \
+RUN sudo dnf install -y \
+        https://copr-be.cloud.fedoraproject.org/results/nmstate/nmstate-git-fedora/fedora-31-x86_64/01067487-nmstate/nmstate-0.0.8-0.20191022.669git5701bbe.fc31.noarch.rpm \
```

Comment:
We should use copr directly, in case the nmstate RPM there has some dependencies. Pleas install the copr addon, configure the repository and then install the RPM with `dnf install -y nmstate-0.0.8-0.20191022.669git5701bbe`

Reply:
This does not work that's why I have put directly the full p ath

```
Fedora 31 - x86_64                               26 kB/s |  17 kB     00:00    
No match for argument: nmstate-0.0.8-0.20191022.669git5701bbe
Error: Unable to find a match: nmstate-0.0.8-0.20191022.669git5701bbe
The command '/bin/sh -c sudo dnf install -y dnf-plugins-core &&     sudo dnf copr enable -y nmstate/nmstate-git-fedora &&     sudo dnf install -y nmstate-0.0.8-0.20191022.669git5701bbe iproute iputils &&     sudo dnf remove -y dnf-plugins-core &&     sudo dnf clea
```

Reply:
Worked for me when I tried it manually. Maybe `install nmstate-0.0.8-0.20191022.669git5701bbe.fc31.noarch`?

Reply:
Yep, working.